actors/builtin/miner/monies.go

```diff
@@ -55,18 +55,17 @@ func PledgePenaltyForUndeclaredFault(epochReward abi.TokenAmount, networkQAPower
 
 // Penalty to locked pledge collateral for the termination of a sector before scheduled expiry.
 // SectorAge is the time between the sector's activation and termination.
-func PledgePenaltyForTermination(initialPledge abi.TokenAmount, sectorAge abi.ChainEpoch, epochTargetReward abi.TokenAmount, networkQAPower, qaSectorPower abi.StoragePower) abi.TokenAmount {
+func PledgePenaltyForTermination(initialPledge abi.TokenAmount, dayReward abi.TokenAmount, sectorAge abi.ChainEpoch, epochTargetReward abi.TokenAmount, networkQAPower, qaSectorPower abi.StoragePower) abi.TokenAmount {
```

Comment:
Name this one `initialDayReward` or `dayRewardAtActivation` to distinguish from one calculated for the current epoch.